core/src/main/java/com/rim/IceField/Map.java

```diff
@@ -6,19 +6,18 @@
 //Map class
 public class Map {
 
+public Iceberg[][] Icebergs = new Iceberg[10][10];
 
-    private ArrayList<Iceberg> icebergs;    //List of icebergs
     private ArrayList<ItemBase> items;      //List of items
 
     //Getter for list of icebergs
-    public ArrayList<Iceberg> getIcebergs() {
-        System.out.println("getIcebergs");
-        return icebergs;
+    public Iceberg[][] getIcebergs() {
+
+        return Icebergs;
```

Comment:
@danilashtefan  Do we use a static array for Iceberg or dynamic? Please, check this with @RainZoneO2, such that you would be in sync

Reply:
I guess that it is static, because icebergs are predetermined from the beginning